### DISCUSSION TITLE
第6节课笔记(9班)

### DISCUSSION THREAD

**Discussion by RangeKing** [InternLM/Tutorial#70] [笔记]
请在下方提交第 6 节课笔记笔记链接，知乎/CSDN/飞书 等等都可。
笔记让知识记得更牢固💪

**Comment by li4692625** [comment]
https://blog.csdn.net/li4692625/article/details/135711683

**Comment by ShelterWFF** [comment]
class6: http://t.csdnimg.cn/IDxSI

**Comment by zhangzhidaSunny** [comment]
https://blog.csdn.net/m0_65719612/article/details/135734314
第六课笔记

**Comment by xiaomile** [comment]
[OpenCompass 大模型评测](https://blog.csdn.net/weixin_42567071/article/details/135737992)

**Comment by KJ-Huang-sz** [comment]
https://blog.csdn.net/qq_40831032/article/details/135729847

**Comment by AIzealotwu** [comment]
https://blog.csdn.net/m0_72806612/article/details/135764154

**Comment by nini0919** [comment]
https://morning-gaura-eed.notion.site/06-336d487d9c234aeeabfab75ab38837e6?pvs=4

**Comment by haizeigh** [comment]
第六课 https://note.youdao.com/s/M9wQMt0d
视频地址 https://www.bilibili.com/video/BV1Gg4y1U7uc/?vd_source=d6b5a09dda93b2c223da08c9b8241a1f
